readthedocs/settings/base.py

```diff
@@ -336,6 +336,14 @@ def USE_PROMOS(self):  # noqa
     CELERYD_PREFETCH_MULTIPLIER = 1
     CELERY_CREATE_MISSING_QUEUES = True
 
+    CELERY_DEFAULT_PRIORITY = 1
```

Comment:
Shouldn't this be 2, so all the tasks go with high priority but only PR with low priority?

Reply:
Or here 1, but 0 on the PR option.